shpc/main/settings.py

```diff
@@ -241,32 +241,6 @@ def change_validate(self, key, value):
                 "%s:%s cannot be added to config: %s" % (key, value, error.message)
             )
 
-    @property
-    def filesystem_registry(self):
-        """
-        Return the first found filesystem registry
-        """
-        for path in self.registry:
-            if path.startswith("http") or not os.path.exists(path):
-                continue
-            return path
-
-    def ensure_filesystem_registry(self):
```

Comment:
I don't remember all the places these were used = but it would be good to do a sanity check grep to ensure it's not needed anymore (I'm away from my main computer now).

Reply:
I used `grep` to find the places it was used, and corrected those.